Sources/SpeziAccount/AccountValue/Configuration/AccountKeyRequirement.swift

```diff
@@ -23,6 +23,10 @@ public enum AccountKeyRequirement {
     /// The account value is **not** collected at signup. However, it is displayed in the account overview
     /// and a user can supply a value by editing their account details.
     case supported
+    /// The associated account value **can** be provided by the user at a later point in time.
+    ///
+    /// The account value is **not** collected at signup. It is also not displayed in the account overview.
+    case hidden
```

Comment:
Thinking about it. Maybe `hidden` is not the best name? After all it might be displayed, you just need to do it yourself. What do you think about calling it `manual` or `selfManaged`? Currently rending to `manual`.
What do you think? Any other ideas?

Reply:
I'm fine with `manual` as well